lxd/main_init_interactive.go

```diff
@@ -127,7 +127,7 @@ func (c *cmdInit) askClustering(config *cmdInitData, d lxd.InstanceServer) error
 
 			s, _, err := d.GetServer()
 			if err == nil {
-				if s.Config["cluster.https_address"] == value || s.Config["core.https_address"] == value {
+				if s.Config["cluster.https_address"] == address || s.Config["core.https_address"] == address {
```

Comment:
Why this change?

As far as I know the value stored in the database is the value as provided by the user, not the value after processing by CanonicalNetworkAddress so we should be checking `value` against the current value and not the updated `address`.

At least that's why I chose to go with value instead of address when I added this last week :)

Reply:
@stgraber I looked into this as was still getting listen errors on subsequent lxd inits after initial lxd init failed part way through for some unrelated reason. 

I found that the output of canonical address is stored in db here https://github.com/tomponline/lxd/blob/5fe4ae42c0c3fd520e2daf31165b6eea8c955157/lxd/main_init_interactive.go#L146 see line above that too.

Reply:
Oh, fun, I checked the server side code, not the client side code :)